src/main/java/org/apache/commons/lang3/ClassUtils.java

```diff
@@ -16,19 +16,11 @@
  */
 package org.apache.commons.lang3;
 
+import org.apache.commons.lang3.mutable.MutableObject;
+
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
-import org.apache.commons.lang3.mutable.MutableObject;
+import java.util.*;
```

Comment:
This won't pass checkstyle I think. You will need to replace the `*` by explicit imports.

See Travis builds failing, this part is due to this change: 

>[ERROR] src/main/java/org/apache/commons/lang3/ClassUtils.java:[23] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - java.util.*.

Reply:
This was the default IntelliJ setting to use * after more than 4 imports from the same package.

I see no reason why not to use '*' in imports these days with such advanced editors, but in case that is the project standard, I comply. Thanks.